README.md

```diff
@@ -4,11 +4,11 @@ The TensorFlow Cloud repository provides APIs that will allow to easily go from
 
 # Introduction
 
-## [`run` API](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/core/README.md)
+- [run API](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/core/README.md)
 
-## [Cloud Tuner](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/tuner/README.md)
+- [Cloud Tuner](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/tuner/README.md)
 
-# `run` API 
+# run API for GCP training/tuning
```

Comment:
Since we're starting to call this "the run API", we need a better, catchier name for this subset of TF Cloud. Thoughts?

Reply:
So i have been referring to this as TF cloud `run` API. TensorFlow Cloud being the catchy term here. I'll update the headers to have the full name for now.